Makefile.am

```diff
@@ -172,7 +172,7 @@ src/ecmult_static_context.h: $(gen_context_BIN)
 CLEANFILES = $(gen_context_BIN) src/ecmult_static_context.h $(JAVAROOT)/$(JAVAORG)/*.class .stamp-java
 endif
 
-EXTRA_DIST = autogen.sh src/gen_context.c src/basic-config.h $(JAVA_FILES)
+EXTRA_DIST = autogen.sh src/gen_context.c src/libsecp256k1-config.h src/basic-config.h $(JAVA_FILES)
```

Comment:
> Can you add a sentence like "Try deleting ecmult_static_context.h before the build." or similar?

Done.

> I don't think we want to distribute src/libsecp256k1-config.h. It will be generated by configure.

That was a misguided attempt at having gen_context rebuild the static context upon configuration change. It seems to work now anyway, so I removed it again.

Reply:
Oh hm, apparently `make distcheck` fails here:
https://travis-ci.org/bitcoin-core/secp256k1/jobs/558864214#L847

On a second thought, I'm not sure that including `src/libsecp256k1-config.h` is a good idea. There are downstream users of this library (e.g., https://github.com/rust-bitcoin/rust-secp256k1) that don't use the autotools. But I'm not sure what the best thing is. Maybe including it only if `ECMULT_GEN_PREC_BITS` is not defined. Sounds pragmatic to me.

Reply:
fixed